flagsmith/models.py

```diff
@@ -13,7 +15,7 @@ class BaseFlag:
     value: typing.Union[str, int, float, bool, None]
 
 
-@dataclass
+@dataclass()
```

Comment:
Not sure why we added the parenthesis here but not below (line 52)

Reply:
Fixed.